javascripts/DOM.js

```diff
@@ -39,3 +39,73 @@ var Chatty = (function(Chatty) {
 })(Chatty || {});
 
 console.log("DOM js", Chatty);
+
+ document.getElementById("input").addEventListener("keypess", function(event) {
+ // console.log("event",event );
+var textInput = document.getElementById("input").value;
+var outputEl = document.getElementById("container");
+    var string = "";
+    var counter = 1;
+  if (event.keyCode === 13) {
+    outputEl.innerHTML = event.target.value;
+      for (var i = 0; i < 100; i++) {
+          string +=
+
+             `<div class="messageContainer">
+              <div class="messageText">
+                <p>${textInput}</p>
+              </div>
+              <div class="deleteButton deleteButton-${counter}">
+                <button>Delete</button>
+              </div>
+            </div>
+          </div>`
+
+      counter++;
+
+};
+
+}
+});
+ 
+
+
+var clear = document.getElementById("clear-button");
+var textInput = document.getElementById("input");
+var messageText = document.getElementById("container");
+
+document.getElementById("textSize").addEventListener("click", function() {
+  messageText.classList.toggle("large");
+});
+
+
+document.getElementById("theme").addEventListener("click", function () {
+  messageText.classList.toggle("change-color");
+})
+
+
+function resetForm() {
+  textInput.value = " ";
+  messageText.innerHTML = " ";
+
+}
+
+clear.addEventListener("click", resetForm);
```

Comment:
got it. I changed the function to be Chatty.clearMessages. then 1. when do those input messages get cleared up from the input bar?
 2. since I moved css to sass/styles.css, the large and change color button doesn't work. what other step am I missing here? I can't think of what to do with Gruntfile